src/cron/main.js

```diff
@@ -4,7 +4,7 @@ const fs = require("fs")
 const dataDir = "/csv/"
 const histDataDir = "/csv/historical/"
 
-croner.Cron("0 */5 * * * *", async () => {
+croner.Cron("0 */1 * * * *", async () => {
```

Comment:
This is the only thing I would consider persisting. Not sure how much of an impact this will have on the VM if we're running every min.

Reply:
I have reverted this to be */1 then.
It sounds like it's optimized enough.

I caught a change where the timer was changed to */42
It didn't seem correct so I adjusted according to your feedback

Thank you!